java/piranha/src/test/java/com/uber/piranha/PiranhaTestingHelpers.java

```diff
@@ -23,4 +23,38 @@ public static BugCheckerRefactoringTestHelper addHelperClasses(
       BugCheckerRefactoringTestHelper bcr) throws IOException {
     return bcr.addInput("XPTest.java").expectUnchanged();
   }
+
+  public static BugCheckerRefactoringTestHelper addExperimentFlagEnumsWithConstructor(
+      BugCheckerRefactoringTestHelper bcr, boolean hasEnumConfiguration) {
+    BugCheckerRefactoringTestHelper.ExpectOutput inputLines =
+        bcr.addInputLines(
+            "TestExperimentName.java",
+            "package com.uber.piranha;",
+            "public enum TestExperimentName {",
+            " STALE_FLAG(\"stale.flag\"),",
+            " OTHER_FLAG(\"other\");",
```

Comment:
Good point. Will add a TODO, as this should probably be a separate fix since I believe Piranha already doesn't handle removal of the last enum constant with the semicolon properly (it only checks for enums ending with a `,` rather than `,` or `;`)

Reply:
Created a ticket https://github.com/uber/piranha/issues/143